### PR TITLE
added values to the y object

### PR DESCRIPTION
**Pull request by claremcarthur**
Hello, this is my attempt to make changes to the practice-repo before I forget what to do 🐵

### CODE REVIEW

**Comment by claremcarthur** [comment]
I seem to have a conflict, think it's because I updated the master copy by mistake first then updated an outdated branch that I had. I'll try again.

File: 01_script.R [hunk 1]
```diff
@@ -3,7 +3,7 @@
 x <- c(1, 2, 3, 4, 5)
 z <- c()
 
-y <- c(4, 7, 8, 9, 10)
+y <- c(4, 7, 8, 9, 10, 15, 22, 25)
```
Comment (ScottMG-PHS) [inline 01_script.R:6]: Test comment

**Review by ScottMG-PHS — COMMENTED** [review]
Just wanted to see how reviews worked

**Comment by ScottMG-PHS** [comment]
> I seem to have a conflict, think it's because I updated the master copy by mistake first then updated an outdated branch that I had. I'll try again.

I think you can pick which line of code you want to keep with the resolve conflicts button?

**Comment by claremcarthur** [comment]
@ScottMG-PHS ready to approve

**Pull request merged**